service_mesh/v2x/installing-ossm.adoc

```diff
@@ -30,4 +30,4 @@ include::modules/ossm-install-ossm-operator.adoc[leveloffset=+1]
 
 == Next steps
 
-Create a `ServiceMeshControlPlane` resource to configure the components of {SMProductShortName}. For more information, see xref:../../service_mesh/v2x/ossm-create-smcp.adoc#ossm-create-smcp[Creating the ServiceMeshControlPlane].
+The {SMProductName} Operator does not create the various {SMProductShortName} custom resource definitions (CRDs) until you deploy a {SMProductShortName} control plane. You use the `ServiceMeshControlPlane` resource to install and configure the {SMProductShortName} components. For more information, see xref:../../service_mesh/v2x/ossm-create-smcp.adoc#ossm-create-smcp[Creating the ServiceMeshControlPlane].
```

Comment:
Can you relook at using the word `various`. I suggest that it can be avoided unless needed.

Reply:
re: "various" Can you think of a more appropriate way to indicate that there are multiple CRDs?

Reply:
plural `CRDs` itself implies multiple. I am unclear regarding the need for having the word. Let me get another pair of eyes to look into this from the squad for an opinion.

Reply:
I think you could omit `various` without losing the meaning of the sentence.

With no extra context, I'm not sure that the next step is clear enough. Does the user already have the control plane at this point? If not, I think it might be two next steps: 
- Deploy the control plane.
- Modify that CRD.

Reply:
This issue came out of a customer who was looking for the Service Mesh CRDs because they want to use source control to manage their configurations.  I checked with engineering and we do things a little differently from standard OLM in that none of the service mesh CRDs exist until you create a control plane (SMCP), creating the control plane is what triggers creating the service mesh CRDs.  If it seems clear that there are multiple because CRDs is plural, then I'm fine with deleting the word "various".